actioncable/lib/action_cable/server/configuration.rb

```diff
@@ -29,7 +29,25 @@ def channel_class_names
           Pathname.new(channel_path).basename.to_s.split('.').first.camelize
         end
       end
+
+      # Returns constant of subscription adapter specified in config/cable.yml.
+      # If the adapter cannot be found, this will default to the Redis adapter.
+      # Also makes sure proper dependencies are required.
+      def pubsub_adapter
+        adapter = (cable.fetch('adapter') { 'redis' })
```

Comment:
I believe that this point of the code is only accessed once; while initializing, so optimizing is not necessary here. But I'm just curious about `fetch` vs block in general